### ISSUE TITLE
Error importing public key from clipboard

### ISSUE THREAD

**Issue by lvl3magik** [open-keychain/open-keychain#2623]
When trying to import public keys from clipboard on Android 10, app says "New keys found, tap to view etc" but when viewed it says no data. Checked all permissions and never had this issue before. 
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Android Version: 
* Device Model: 
* OpenKeychain Version: 
* From Google Play or F-Droid?:

**Comment by PGPA20e** [comment]
Yes,  It's the same thing for me, I cannot import new key from clipboard and its very annoying.

**Comment by str8outlaw** [comment]
How do i fix this issue ??? Its definitley nit allowing me to import keys from clipboard.

**Comment by str8outlaw** [comment]
Its beyond annoying

**Comment by lvl3magik** [comment]
> Its beyond annoying

So I've figured out a work around for the time being. 

1. Copy public PGP key to clipboard. 
2. Go to the "KEYS" subdirectory and tap the "+" like you're going to add a key etc.
3. Go to "Import from File".
4. A new window will pop up saying "Import Keys".
5. In the upper right hand side tap the 3 vertical dots and select "Import from Clipboard". 

There you go. That's how I do it. The development team obviously isn't too concerned with this bug or they would've fixed it now. 

Hope that helps!

**Comment by lvl3magik** [comment]
> Yes, It's the same thing for me, I cannot import new key from clipboard and its very annoying.

> Its beyond annoying

So I've figured out a work around for the time being. 

1. Copy public PGP key to clipboard. 
2. Go to the "KEYS" subdirectory and tap the "+" like you're going to add a key etc.
3. Go to "Import from File".
4. A new window will pop up saying "Import Keys".
5. In the upper right hand side tap the 3 vertical dots and select "Import from Clipboard". 

There you go. That's how I do it. The development team obviously isn't too concerned with this bug or they would've fixed it now. 

Hope that helps!

**Comment by lvl3magik** [comment]
> When trying to import public keys from clipboard on Android 10, app says "New keys found, tap to view etc" but when viewed it says no data. Checked all permissions and never had this issue before.
> 
> ## Expected Behavior
> ## Current Behavior
> ## Possible Solution
> ## Steps to Reproduce (for bugs)
> 
> ## Context
> ## Your Environment
> * Android Version: 10.0
> * Device Model: Samsung A10E
> * OpenKeychain Version: Newest
> * From Google Play or F-Droid?: Google Play, tried F-Droid as well same issue.
>   ![Screenshot_20201214-132233_OpenKeychain](https://user-images.githubusercontent.com/76009091/102125919-fd4c4e00-3e0f-11eb-9717-a8c38e071b99.jpg)
>   ![20201214_132437](https://user-images.githubusercontent.com/76009091/102125922-fde4e480-3e0f-11eb-90ad-2a786a871436.jpg)

I found a workaround for this. 

> Its beyond annoying

So I've figured out a work around for the time being. 

1. Copy public PGP key to clipboard. 
2. Go to the "KEYS" subdirectory and tap the "+" like you're going to add a key etc.
3. Go to "Import from File".
4. A new window will pop up saying "Import Keys".
5. In the upper right hand side tap the 3 vertical dots and select "Import from Clipboard".
